tests/baselines/reference/jsdocSignatureOnReturnedFunction.js

```diff
@@ -0,0 +1,50 @@
+//// [jsdocSignatureOnReturnedFunction.js]
+function f1() {
+    /**
```

Comment:
can you add a test with `@type` as well?

Reply:
I've added tests with `@type`